viite-UI/src/model/RoadAddressProjectCollection.js

```diff
@@ -135,22 +139,29 @@
 
       projectLinksSaved = projectLinksSaved.concat(linkIds);
 
-      var projectId = currentProject.id;
+      var projectId = projectinfo.id;
 
       var data = {'linkIds': linkIds, 'projectId': projectId, 'newStatus': STATUS_TERMINATED};
 
+      console.log(data);
+      console.log(linkIds);
       if(!_.isEmpty(linkIds) && typeof projectId !== 'undefined' && projectId !== 0){
         backend.updateProjectLinks(data, function(errorObject) {
           if (errorObject.status == INTERNAL_SERVER_ERROR_500 || errorObject.status == BAD_REQUEST_400) {
             eventbus.trigger('roadAddress:projectLinksUpdateFailed', errorObject.status);
+          } else {
+            console.log("TODO Success, refresh now");
```

Comment:
Dirty data means that the data is modified in the UI but not yet saved to the database. After successful save we should reload the road links from the backend to bring the latest view.